server/service/queries_test.go

```diff
@@ -356,6 +254,12 @@ func TestQueryAuth(t *testing.T) {
 		ID:   1,
 		Name: "Foobar",
```

Comment:
the following additions are adding test cases to ensure Team Users can't view queries on other teams